README.md

```diff
@@ -681,3 +437,120 @@ thereafter it should be possible to use one of the different checkpatch targets
 in the [Makefile](Makefile). There are targets for checking all files, checking
 against latest commit, against a certain base-commit etc. For the details, read
 the [Makefile](Makefile).
+
+## 6. repo manifests
```

Comment:
In that case, what about:

* leaving @jforissier changes made here
* have https://github.com/OP-TEE/manifest/blob/master/README.md indicates instruction to use git manifest can be found in optee_os/readme[section 6]?

or (second option): we have duplicated instructions till GP conference is over?